backend/controllers/authController.js

```diff
@@ -0,0 +1,64 @@
+const User = require("../models/User");
+
+const handleErrors = (err) => {
+  if (err.code === 11000) {
+    return { email: 'Email is already registered' };
+  }
+return  Object.keys(err.errors).reduce((acc, key) => {
+    acc[key] = err.errors[key].message;
+  return acc;
+}, {});
+}
```

Comment:
this method checks error codes and errors to build out custom messages for them. Uses reduce to build out object for multiple errors.